svelte.config.js

```diff
@@ -7,6 +8,7 @@ const extensions = [`.svelte`, '.md', `.mdx`, '.svx'];
 /** @type {import('@sveltejs/kit').Config} */
 const config = {
 	preprocess: [
+		preprocess(),
```

Comment:
i'm curious why this had to be added

Reply:
Without it, all `*.svelte` files that use `lang="ts"` are not convert from Typescript to Javascript.

So it is need for:
 - `src/lib/components/layout/Header.svelte`
 - `src/routes/recipes/__layout.svelte`
 - `src/routes/events/index.svelte`
 - `src/routes/cheatsheet/_CheatSheetCard.svelte`

But it can be remove if we "convert" (just some `import type` and very few typing to remove to plain JS.

It can also be more precisely defined:
```js
// ...
import { typescript } from 'svelte-preprocess';
// ...
	preprocess: [
		typescript(),
// ...
```

Reply:
I think it's fine to leave this in. Ideally we'd want this to be 100% TS.